### ISSUE TITLE
Update GWLF-E Coefficients

### ISSUE THREAD

**Issue by rajadain** [WikiWatershed/model-my-watershed#3526]
From BME:

> I'm done with my calibration work, and Mike will be updating the sub-basin API to make the necessary changes in the attenuation factors. As you know, you will not need to do anything on your end for this update. However, as you probably know, users of MMW also have the option for doing a simple "manual" attenuation process by adjusting certain factors in the gms file as shown in the attached figure. Right now we have suggested attenuation factors of 0.12 (for N), 0.29 (for P), and 0.84 (for TSS). When you have a moment, could you please change these to 0.06 (for N), 0.45 (for P), and 0.95 (for TSS)? No rush on this; let me know if you have any questions. Thanks much.

![ManualCoeffs](https://user-images.githubusercontent.com/1430060/163594438-84a52e4d-fa54-496a-acb5-0348fbfbfbc1.JPG)

These coefficients are defined here:

https://github.com/WikiWatershed/model-my-watershed/blob/84aad7632fa62f68b9ac66dee909534550485e8c/src/mmw/mmw/settings/gwlfe_settings.py#L436-L438

- [x] Change those coefficients to the new values defined above
- [ ] Add a migration to update all existing scenarios that don't already override that coefficient to set it to the old values. This is similar to what was previously done in #3448

**Comment by emilyhu0106** [comment]
I'm suspecting that simply updating the coefficient values is enough to address this issue. My understanding for our goal is using new values for new projects, while keeping the old values for existing projects, including new scenarios created from old projects. My investigation is detailed below:

When the coefficients are not changed, aka 0.12 (for N), 0.29 (for P), and 0.84 (for TSS), I created Watershed Multi-Year Projects **Project 1** with two scenarios: 
 1. **Scenario default**: Using the default value
 2. **Scenario 0.9**: Updating the values to 0.9 (for N), 0.9 (for P), and 0.9 (for TSS)
 

Then I updated the coefficient to the new value and restarted celery
```
    'RetentNLake': 0.06,  # Lake Retention Rate: Nitrogen
    'RetentPLake': 0.45,  # Lake Retention Rate: Phosphorus
    'RetentSedLake': 0.95,  # Lake Retention Rate: Sediment
```

I created Watershed Multi-Year Projects **Project 2** and add a new scenario. The coefficient is the new value, which is expected
![image](https://user-images.githubusercontent.com/60887686/166330564-1d561787-b3e2-4358-9c81-971d9a308632.png)


Now, back in **Project 1 - Scenario default**, the coefficients are still the old value:
![image](https://user-images.githubusercontent.com/60887686/166330933-53132541-c830-4310-a058-9c83064f43f6.png)

In **Project 1 - Scenario 0.9**, the updated values - 0.9 (for N), 0.9 (for P), and 0.9 (for TSS) - can be reset to the old value
![2022-05-02 14 33 11](https://user-images.githubusercontent.com/60887686/166331261-0464e582-06b1-416b-8458-757c864768e9.gif)

In **Project 1**, create a new scenario, and the displayed coefficients are still the old value:
![image](https://user-images.githubusercontent.com/60887686/166331384-377ae590-1251-4f6c-85a5-baa3e224700f.png)
